.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug 反馈
+about: 通过该 Issue 模板反馈使用 tongji-undergrad-thesis 时的 bug
+labels: 'type/bug'
+---
+
+### 使用平台详情
+
+<!--在你所使用的平台上打勾，或填写你使用的平台-->
+
+* [ ] 线上：Overleaf
+* [ ] 本地：MacOS
+* [ ] 本地：Windows
+* [ ] 本地：各种 Linux 发行版
+* [ ] 不，我使用的是 _________
+
+<!--此外，建议你在下方详细描述你所使用的工具，如编辑器与编译器类型等等-->
+
+### 编译日志
+
+<!--在下方贴出编译时输出的错误日志-->
+
+```log
+
+```
```

Comment:
```suggestion
<details><summary><b>报错内容</b> <i>[点击打开]</i></summary>
<div>

```log
# Paste your log here:
\```

</div>
</details>
```

Reply:
The point is, you don't want the long long bug logs pollute your page into endless bottom.

Reply:
Nice! This is awesome.